src/Microsoft.ApplicationInsights/Extensibility/TelemetryConfiguration.cs

```diff
@@ -130,6 +132,23 @@ public string InstrumentationKey
             }
         }
 
+        /// <summary>
+        /// Sets an override to "https://dc.services.visualstudio.com".
+        /// All of our internal services will construct the full endpoint address starting with this base address.
```

Comment:
Except for QuickPulse, right?

Reply:
Yes, except for QuickPulse. Is there a more formal name for this service?